tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author ${author} - Initial contribution
  */
-@Component(service = ThingHandlerFactory.class, immediate = true)
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPolicy = ConfigurationPolicy.OPTIONAL, name = "binding.${bindingId}")
```

Comment:
Now as you say it, I overlooked that it is default. I guess I copied it from another service that had it specified. I fixed the archetype, see PR #4125